Library/Homebrew/cmd/test-bot.rb

```diff
@@ -981,4 +983,35 @@ def test_bot
 
     Homebrew.failed = any_errors
   end
+
+  def sanitize_output_for_xml(output)
+    unless output.empty?
```

Comment:
I don't know if that's Homebrew's house style? I personally like to take early returns to keep indentation/nesting low too, but most of the `brew` code base doesn't seem to do that. Is there some guidance on that?